### PR TITLE
Simplified set operations

### PR DESCRIPTION
**Pull request by AlexanderSaydakov**
removed NewDegen state, renamed New to Empty, some rewording and rearrangement

### CODE REVIEW

File: docs/Theta/ThetaSetOpsCornerCases.md [hunk 1]
```diff
@@ -82,60 +79,38 @@ We can assign a single octal digit ID to each state where
...
+| Estimation {<1.0,>0,F}            | <1.0  |        >0        |     F      |       T       |  2 | Estimation Mode                 |
+| Degenerate {<1.0,0,F}<sup>3</sup> | <1.0  |         0        |     F      |       T       |  0 | Valid Intersect or AnotB result |
 
 ---
 
-<sup>2</sup> *New Degenerate*: New Empty Sketch where the user sets *p < 1.0*. This can be safely reinterpreted as {1.0,0,T} because it has not seen any data.<br>
-<sup>3</sup> *Result Degenerate*: Can appear as a result of a an Intersection or AnotB of certain combination of sketches.
-
-## Invalid States of a Sketch
-The remaining three combinations of the variables represent internal errors and should not occur.
```
Comment (leerho) [inline docs/Theta/ThetaSetOpsCornerCases.md:99]: Why did you remove the clarification of the invalid states of the sketch?  This is important because the average reader will immediately know that 3 boolean values yield 8 possible combinations and wonder why don't we discuss those corner cases!  Just because it is obvious to you doesn't mean it will be obvious to the reader why we can ignore the other 4 cases.

**Review by leerho — CHANGES_REQUESTED** [review]

File: docs/Theta/ThetaSetOpsCornerCases.md (hunk 1 above)
Comment (AlexanderSaydakov) [reply docs/Theta/ThetaSetOpsCornerCases.md:99]: I don't see any merit in discussing invalid combinations. We clearly state that out of possible 8 combinations only 4 are valid. So I thought why should we clutter the document with unnecessary stuff.

Comment (leerho) [reply docs/Theta/ThetaSetOpsCornerCases.md:99]: 1. Explaining _why_ the other 4 cases are invalid is instructive.  It helps the user understand what is going on inside the sketch.
2. It makes the analysis complete.
3. It conveys confidence to the reader that we have thought through all the cases.
4. It is only about 10 lines of text including a simple table out of around 160 lines in the document,  hardly a distraction and not what I would call unnecessary clutter.

Comment (AlexanderSaydakov) [reply docs/Theta/ThetaSetOpsCornerCases.md:99]: OK, I restored this section. The "empty flag" column in the table was wrong, so I corrected it. I am not convinced this is needed. It does not really explain _why_. These are just invalid states. One of them, which was the NewDegen before, is used in practice as the initial state of the update sketch with p < 1, but this is an implementation detail, and is not visible from the outside. I am not sure we need to explain this, and it was a part of my motivation to remove this section. If it is not very useful, I would call it clutter.

Comment (leerho) [reply docs/Theta/ThetaSetOpsCornerCases.md:99]: I agree that the empty flag column was inverted.  Apparently you also misunderstood my use of the "!=" symbol. My use of "!=" was to say "cannot be equal to".  and "!<" meant "cannot be less than", etc.  So I spelled this out in words to be more clear and submitted the change.  With these clearer comments it should be obvious _why_ these cases are invalid.  :)

**Review by leerho — APPROVED** [review]

File: docs/Theta/ThetaSetOpsCornerCases.md (hunk 1 above)
Comment (leerho) [reply docs/Theta/ThetaSetOpsCornerCases.md:99]: If you approve, mark your approval, then you can merge.

Comment (AlexanderSaydakov) [reply docs/Theta/ThetaSetOpsCornerCases.md:99]: I cannot approve my own pull request. I agree with your changes. I would prefer not to have this section at all, but I don't think this is very important. Let's merge.
